### DISCUSSION TITLE
Notify users about quality of Martin fonts

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/martin#1155] [Q&A]
Should we notify users about the quality of fonts served with Martin? I think many will not be aware that fonts generated with Martin are more pixelated than the ones generated with font-maker...

**Comment by nyurik** [comment]
@wipfli I am not certain why this might be the case - shouldn't Martin generate the same font types, with the same resolution?  Martin uses [pbf_font_tools](https://crates.io/crates/pbf_font_tools) by @lseelenbinder and @ianthetechie which internally handles all that.

**Reply by ianthetechie** [reply]
Yeah, nobody has ever complained to us about it yet, but TL;DR the crate currently uses the methodology from TinySDF whereby it rasterizes to a bitmap first and generates the SDF from that. This is faster but slightly lower quality. 

There is an open issue to add support for the other method. PRs welcome; it’s not hard to port the C++ code. Just hasn’t been high on my priority queue. https://github.com/stadiamaps/sdf_font_tools/issues/16
